user_created.py

```diff
@@ -0,0 +1,73 @@
+
+
+import datetime
+import json
+import sys
+import time
+
+import ddb
+import configuration
+import utils
+
+
+class UserCreated(object):
+    table_name = "UserCreated"
+
+    def __init__(self, fake=False):
+        self.ddb = ddb.DDB(self.table_name, [('slack_uid', 'S')])
+        self.table = self.ddb.get_table()
+        self.users = {}
+        self.new = []
+        self.fake = fake
+
+    def date(self, ts):
+        """
+        return yyyy-mm-dd for ts
+        """
+        localtime = time.localtime(ts)
+        return time.strftime("%Y-%m-%d", localtime)
+
+    def days(self, ts):
+        """
+        return date difference between today and the ts
+        """
+        then = datetime.datetime.fromtimestamp(ts)
+        now = datetime.datetime.now()
+        diff = now - then
+        return diff.days
+
+    def load(self):
```

Comment:
Will you please write a docblock for this?

Reply:
Sorry, for load()?

Reply:
Yep. Thanks. What you wrote in the docblock makes perfect sense.

Reply:
Added a docblock for load()